types/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
its an interesting choice to essentially deliver the types in a separate package. i'm not certain what the advantages are to this approach over simply adding an `index.d.ts` to the root of the package. can you explain? one disadvantage i see is that its common practice to list the `@types/*` dependencies in the top-level `package.json`, [see this section of the manual](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies).

is the intention that we would publish two separate modules to npm? if so, i think that would be confusing, and at that point we might as well only offer ambient declarations via DefinitelyTyped.

Reply:
[Typescript's publishing documentation](http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) suggests publishing to the @types organization on npm if the package is not written in TypeScript.

I believe the intent is to avoid shipping type definitions to those who don't need them.

That said, I'm happy to instead add `index.d.ts` if that is your preferred approach.